dl/src/main/scala/com/intel/analytics/bigdl/utils/RandomGenerator.scala

```diff
@@ -183,6 +183,9 @@ class RandomGenerator() {
     this
   }
 
+  /**
```

Comment:
@linbojin We just check the style in the build. To do code format, I personally use IDE reformat function...